bin/cli.js

```diff
@@ -265,7 +266,7 @@ function createOutputFromOptions(program, res) {
 			json: program.json
 		});
 
-		if (circular.length) {
+		if (circular.length && !program.allowFailures) {
```

Comment:
```suggestion
		if (!program.allowFailures && circular.length) {
```

Kinda niche-picking but in this way it express better which is more important